features/mesh/dataplanes/DataplaneDetailsTraffic.feature

```diff
@@ -77,33 +77,3 @@ Feature: Dataplane traffic
     When I visit the "/meshes/default/data-planes/dpp-1-name-of-dataplane/overview" URL
     And the "$traffic" element exists
     And the "$loading-warning" element exists
-
-  Scenario: Builtin gateway proxy doesn't show the traffic component
-    Given the URL "/meshes/default/dataplanes/dpp-1-name-of-gateway_builtin/_overview" responds with
-      """
-      body:
-        dataplane:
-          networking:
-            gateway:
-              type: BUILTIN
-      """
-
-    When I visit the "/meshes/default/data-planes/dpp-1-name-of-gateway_builtin/overview" URL
-
-    And the "$detail-view" element contains "dpp-1-name-of-gateway_builtin"
-    And the "$traffic" element doesn't exist
-
-  Scenario: Delegated gateway proxy doesn't show the traffic component
-    Given the URL "/meshes/default/dataplanes/dpp-1-name-of-gateway_delegated/_overview" responds with
-      """
-      body:
-        dataplane:
-          networking:
-            gateway:
-              type: DELEGATED
-      """
-
-    When I visit the "/meshes/default/data-planes/dpp-1-name-of-gateway_delegated/overview" URL
-
-    And the "$detail-view" element contains "dpp-1-name-of-gateway_delegated"
-    And the "$traffic" element doesn't exist
```

Comment:
This are no longer required. I made an issue about that fact that these were passing also

https://github.com/kumahq/kuma-gui/issues/2350